corehq/apps/app_manager/views/forms.py

```diff
@@ -755,6 +754,10 @@ def is_valid(value):
         }
     else:
         refs = json.loads(data.get('case_references', '{}'))
-    if set(refs) - {"load"} or not all(is_valid(v) for v in refs["load"].values()):
+
+    try:
+        references = CaseReferences.wrap(copy.deepcopy(refs))
```

Comment:
actually i thought this comment was somewhere else, but removed it here too. https://github.com/dimagi/commcare-hq/pull/14710/commits/ca63db1e42a26cb806768da474174457b860b6e4